diwako_cbrn.vr/scripts/cbrn/functions/fn_detectorGeigerPFH.sqf

```diff
@@ -0,0 +1,35 @@
+if (!isGameFocused || isGamePaused) exitWith {};
+params ["_args", "_idPFH"];
+_args params ["_lastIteration"];
+
+private _time = cba_missiontime;
+private _delta = _time - _lastIteration;
+_args set [0, _time];
+
+if ( !cbrn_geiger ||  {!(ace_player getVariable ["cbrn_detector_geiger", false])} ) exitWith {
+    cbrn_geigerPfh = -1;
+    [_idPFH] call CBA_fnc_removePerFrameHandler;
+};
+
+if (cbrn_curThreat < 0.25) exitWith {};
+
+private _wait = 2;
```

Comment:
Personally I do not see a point in writing a variable as writing variable takes (albeit a short) time. if the value is a constant as well, why not use the value itself directly without writing it into a variable?